### ISSUE TITLE
Posts featured images not shown on front office

### ISSUE THREAD

**Issue by tidepccsa** [smartdatasoft/smartblog#119]
Good morning,

I use the 3.0 release on Prestashop 1.7.5.1 and the featured images of the blog posts are not shown in front office, but they are uploaded succesfully and shown correctly in the back office, specifically in the post creation menu.

The only way I found to avoid this issue is to disable mod_rewrite of the URLs, but I would prefer not to do so if possible.

Thank you for your attention and I hope that someone can help me in this matter

**Comment by thierry-laval** [comment]
I have this probleme also... And a problem with debog mod

**Comment by KoveZoli** [comment]
This may be a solution: allow_url_fopen ON

**Comment by sirius82** [comment]
Hello, I've the same problem. Didi you find a solution ? allow_url_fopen is on. I tried without mod rewrite. But it's always the same.
Thanks

**Comment by sirius82** [comment]
Solved : re-initialize the module after tables and images backup. Unable and enable SEO. Re-import tables and images.

**Comment by LookSkarpeta** [comment]
Unfortunately I have the same issue. All seems to be OK but feature images are not shown in the store. The only option to see pictures was to disable friendly URL in whole store but this is not an option. I installed module again, I changed url showing format from xx_xx to xx-xx. Do somebody have a solution? Having blog without images looks poor and not attractive at all : https://skarpetoholik.pl/smartblog/12_skarpetki-trendy-na-rok-2020-.html

PS 1.7.5.0
SmartBlog 3.0
